### DISCUSSION TITLE
Add an option to populate fields

### DISCUSSION THREAD

**Discussion by ajomuch92** [pocketbase/pocketbase#1165] [Q&A]
There are case when we want to get a children list only getting accesing throght the parent (e.g. get an order and get its details on same request), for this, now, we have to make two request, one for the parent and other for children. Expand option allows to get the parent item on a relation, but no children.

If there is an option for this, let us know

**Comment by ganigeorgiev** [comment]
Nested relations are allowed to be expanded using dot-notation (eg. `relField.subRelField`) as long as the request client has "View API rule" access to the rel collections. With v0.8.0, we also support "Indirect expand", aka. expand where the relation field is not in the main (parent) collection.

You can find more info for `expand` in https://pocketbase.io/docs/expanding-relations/.
